Gemfile.lock

```diff
@@ -182,64 +182,62 @@ GEM
       addressable (~> 2.7)
     letter_opener (1.7.0)
       launchy (~> 2.2)
-    listen (3.4.1)
+    listen (3.5.1)
       rb-fsevent (~> 0.10, >= 0.10.3)
       rb-inotify (~> 0.9, >= 0.9.10)
-    loofah (2.9.0)
+    loofah (2.9.1)
```

Comment:
### [loofah](https://github.com/flavorjones/loofah)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.9.0 → 2.9.1

[_[change-log](https://github.com/flavorjones/loofah/blob/main/CHANGELOG.md), [source-code](https://github.com/flavorjones/loofah)_]





<details>
<summary>Commits</summary>

A change of **8** commits. See the full changes on [the compare page](https://github.com/flavorjones/loofah/compare/v2.9.0...v2.9.1).

These are the individual commits:
- (82f0c9e) [ci: update ruby versions tested](https://github.com/flavorjones/loofah/commit/82f0c9e69ba02a8fbb35c459bef0c0cf26db6759)
- (07e89a5) [test: extend Nokogiri decorator tests to NodeSets](https://github.com/flavorjones/loofah/commit/07e89a5794b882c82f3d0fea39252e56948936e7)
- (cbbb3ac) [dev: update default branch to `main`](https://github.com/flavorjones/loofah/commit/cbbb3acd1f4cd4d917c482c4bc1978648185e185)
- (1178f68) [readme: fix CI badge](https://github.com/flavorjones/loofah/commit/1178f6854ea9f2391d8380321f5ff8b67fd06d7f)
- (cc5ce1c) [readme: convert an old rdoc header to markdown](https://github.com/flavorjones/loofah/commit/cc5ce1c6ade918a03730c83cb870e968b7598120)
- (1565bfe) [fix: allow CSS properties to have quoted string values](https://github.com/flavorjones/loofah/commit/1565bfea2e7246bbf1dd73ee8d7028a3f9eb141d)
- (bbde91a) [Merge pull request #203 from flavorjones/202-property-string…](https://github.com/flavorjones/loofah/commit/bbde91a6abe9b71a99511351760356cd6f1b20db)
- (4a6be02) [version bump to v2.9.1](https://github.com/flavorjones/loofah/commit/4a6be028e32b24fe543e0d77c4a998e61ae7a9b5)


</details>